src/single-spa-react.js

```diff
@@ -26,14 +26,17 @@ try {
 const defaultOpts = {
   // required opts
   React: null,
+
+  // required - one or the other
   ReactDOM: null,
+  ReactDOMClient: null,
 
   // required - one or the other
   rootComponent: null,
   loadRootComponent: null,
 
   // optional opts
-  renderType: null,
+  renderType: "createRoot",
```

Comment:
Should this be considered a breaking change?

Reply:
Hmm, yeah, probably, since it changes a lot of how React itself renders things.